### PR TITLE
Pull Request created

### PR DESCRIPTION
**Pull request by Petliuk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,14 +5,14 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>3.4.1</version>
+        <version>3.4.0</version>
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
-    <groupId>com.example</groupId>
-    <artifactId>mate_academy</artifactId>
+    <groupId>mate.academy</groupId>
```
Comment (liliia-ponomarenko) [inline pom.xml]: Don’t  use mate.academy in your project

File: pom.xml [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
+    <artifactId>intro</artifactId>
     <version>0.0.1-SNAPSHOT</version>
-    <name>mate.academy</name>
-    <description>mate.academy</description>
+    <name>into</name>
+    <description>into</description>
```
Comment (liliia-ponomarenko) [inline pom.xml:15]: add checkstyle plugin and setup ci-cd

File: src/main/java/com/example/mate_academy/controller/BookController.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/books")
+public class BookController {
+
```
Comment (liliia-ponomarenko) [inline src/main/java/com/example/mate_academy/controller/BookController.java]:
```suggestion
```

File: src/main/java/com/example/mate_academy/controller/BookController.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import com.example.mate_academy.service.BookService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/books")
+public class BookController {
```
Comment (liliia-ponomarenko) [inline src/main/java/com/example/mate_academy/controller/BookController.java]: Your task doesn't include web implementation. Let's follow the task requirements.

File: src/main/java/com/example/mate_academy/model/Book.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+@Data
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String title;
+    private String author;
+    private String isbn;
+
```
Comment (liliia-ponomarenko) [inline src/main/java/com/example/mate_academy/model/Book.java]: not all entity fields are declared

File: src/test/resources/application.properties [hunk 6]
```diff
@@ -0,0 +1,4 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/db?serverTimezone=UTC
```
Comment (liliia-ponomarenko) [inline src/test/resources/application.properties]: Let’s  use h2 for test configuration

**Review by liliia-ponomarenko — CHANGES_REQUESTED** [review]
Good job! Let’s complete your solution ;)

**Review by sarakhmen — APPROVED** [review]

**Pull request merged**